README.md

```diff
@@ -209,6 +209,24 @@ While it's an implementation detail, the new REST API uses many Spring REST (Jav
 
 Each of these libraries were chosen based on design principles above, and based on the fact that much of our underlying Java APIs use or align with Spring technologies.
 
+## Content negotiation
+The DSpace REST API only support the application/json response format at the current stage so there is no real content negotiation according to the broad meaning of the term.
+Nevertheless, some concepts of content negotiation still apply.
+For the POST requests the request content-type usually drive the creation of resource process according to different input format other than the preferred application/json such as text/uri-list and multipart/form-data. PUT requests on association endpoint accepts text/uri-list according to the Spring Data best practice.
+
+### Language supports
+While most i18n (internationalization) settings exist in the UI layer, there are some backend features which also require i18n, such as Submission forms, Controlled Vocabularies, Authorities. For such features the **Accept-Language** header can be used by the REST client to force the use of a specific locale, if supported by the DSpace instance and endpoint in the response. Please note that not all the dspace instances support multiple locales, this can be discovered querying the configuration endpoints for the _webui.supported.locales_ key.
+
+If the support for multiple locales is enabled in the DSpace instance, a request with an **Accept-Language** header is expected to be processed as follow
+* if none of the requested locales is supported the server will ignore the header to result more user friendly to whom  browses the REST endpoints via a browser (see https://tools.ietf.org/html/rfc7231#section-5.3.5)
```

Comment:
This sentence doesn't make sense in English. I think you mean to say:
"if none of the requested locales is supported, the server will ignore the header.  This provides more user friendly behavior when browsing the REST API via a browser (see https://tools.ietf.org/html/rfc7231#section-5.3.5)."

I was going to make this change myself, but it looks like you've disabled edit privileges for others on this PR

Reply:
thanks for the help here. I don't know why edit privileges are missing and it doesn't seem possible to me to add them... maybe there are some enforced security on our organisation that prevent this functionality to work anymore? I will check more closely when I will open the next PR